extension/src/experiments/webview/contract.ts

```diff
@@ -6,8 +6,13 @@ export interface MetricOrParamColumns {
   [filename: string]: ValueTree
 }
 
+export interface ValueWithChanges {
+  value: string | number
```

Comment:
Could this ever be anything else than a string or number? I think we have a similar interface somewhere and we also have `string[]`, `number[]`,  `boolean`, `boolean[]` in there.